OneSignalExample/Packages/com.onesignal.unity.core/Runtime/Core/IOneSignalPlatform.cs

```diff
@@ -0,0 +1,66 @@
+using System.Collections.Generic;
```

Comment:
Needs to be updated to match changes from master
https://github.com/OneSignal/OneSignal-Unity-SDK/blob/master/OneSignalExample/Assets/OneSignal/src/OneSignal.cs